container.go

```diff
@@ -96,7 +96,7 @@ type Container interface {
 	//
 	// errors:
 	// Systemerror - System error.
-	Set() error
+	Set(config configs.Config) error
```

Comment:
This gives me the assumption that the API allows us to change any of the configuration, while it probably only makes sense to change the cgroup limits (and possibly rlimits ).

Do you think we should define a subset of configuration (i.e. cgroup values, rlimits etc. ) that can be changed for a running container instead of reusing `Config` here ?